config/prow/cluster/trusted_serviceaccounts.yaml

```diff
@@ -46,6 +46,14 @@ metadata:
     iam.gke.io/gcp-service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod-bak.iam.gserviceaccount.com
   name: k8s-infra-gcr-promoter-bak
   namespace: test-pods
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: k8s-infra-gcr-vuln-dashboard@k8s-artifacts-prod.iam.gserviceaccount.com
+  name: k8s-infra-gcr-vuln-dashboard
+  namespace: test-pods
```

Comment:
Does the below comment apply to this service account as well?
> ...accounts.sh on the above

Reply:
Yes and no. If you look at `workload-identity/README.md` it says this:

```
### GCP service account

Once this service account exists in the cluster, then an owner of
`SOMEBODY@PROJECT.iam.gserviceaccount.com` -- typically a `PROJECT` owner --
runs `bind-service-accounts.sh`:
```

So "yes" if we follow the above directions. However we already have this sort of logic tucked away in https://github.com/kubernetes/k8s.io/blob/master/infra/gcp/ensure-prod-storage.sh#L368 which does the binding via https://github.com/kubernetes/k8s.io/blob/f0db0a083f03b443781e3e835150dbc6561fc763/infra/gcp/lib.sh#L498. So in our case, "no," we don't have to.